src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +48,21 @@
     }
   }
 
+  /**
+   * Get the historical price data for the passed in symbol over the passed in range.
+   *
+   * @param symbol stock symbols to get historical data for.
+   * @param range selection of time for which to get historical data.
+   * @return a list of historical price objects for the Symbol that is passed in covering the date
+   *     range.
+   */
+  public List<IexHistoricalPrice> getHistoricalPricesForSymbol(final String symbol,
+      final String range, final String token) {
+    if ("".equals(symbol) || "".equals(range) || "".equals(token)) {
```

Comment:
What if these values are null? Do we want to return an empty list if the range is not provided?